src/core/services/managers/marker-manager.ts

```diff
@@ -61,6 +61,12 @@ export class MarkerManager {
     return this._markers.get(marker).then((m: Marker) => m.setZIndex(marker.zIndex));
   }
 
+  updateAnimation(marker: SebmGoogleMapMarker): Promise<void> {
+    return this._markers.get(marker).then(
+        (m: Marker) =>
+            this._mapsWrapper.getAnimation(marker.animation).then((anim) => m.setAnimation(anim)));
```

Comment:
With this way, we can get rid of the getAnimation method and support all animations in the future.
m.setAnimation(google.maps.Animation[marker.animation])